src/app/authentication/commons/constants/errors.go

```diff
@@ -1,3 +1,8 @@
 package constants
 
-// Add your error constants here
+import "errors"
+
+var (
+	ErrOtpVerification = errors.New("OTP verification failed")
```

Comment:
define variable as string only.
use erros.New while returning, to keep consistency as other's have done the same.